app/views/layouts/application.html.erb

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>BudgetApp</title>
+    <meta name="viewport" content="width=device-width,initial-scale=1">
+    <%= csrf_meta_tags %>
+    <%= csp_meta_tag %>
+
+    <%= stylesheet_link_tag "application", 'style', "data-turbo-track": "reload" %>
+    <%= javascript_importmap_tags %>
+  </head>
+
+  <body>
+  <p class="notice"><%= notice %></p>
+  <p class="alert"><%= alert %></p>
+
+   
+    <%= yield %>
+  </body>
```

Comment:
- [ ] Please consider fixing your UI. All of your views have a blank space at the top of the screen:

![image](https://github.com/eudondian/Budget-App/assets/77069011/44e5b168-5fcc-47b2-9ef6-71278588ef95) 

Remember that is a requirement following as closely as possible the given design guidelines. Please fix it! 🤓 🔧